### PR TITLE
Add recommendation for cached-property

### PR DESCRIPTION
**Pull request by pipermerriam**
Add `cached-property` recommendation

### CODE REVIEW

File: go-to-libraries.md [hunk 1]
```diff
@@ -37,3 +37,9 @@ library doesn't appear to be very well maintained or documented and there are
 places where the documentation doesn't match the library or the library doesn't
 behave in documented ways.  Expect to end up reading the code some if you
 choose this for anything very complex.
+
+### [`cached-property`](https://pypi.org/project/cached-property/)
+
+@piper
```
Comment (ralexstokes) [inline go-to-libraries.md]: not sure why this is here

Comment (pipermerriam) [reply go-to-libraries.md]: Yeah, I figure it's good to know who was championing something.  In theory git blame works for this but that's bad UX.

Comment (carver) [reply go-to-libraries.md]: `@piper suggests:`
or
`@piper recommends:`
to avoid future questions of the same type :)

**Comment by pipermerriam** [comment]
FYI about `cached_property` and slots:  https://github.com/pydanny/cached-property/issues/69

Looks like there's a work-around...

**Comment by carver** [comment]
> FYI about `cached_property` and slots: [pydanny/cached-property#69](https://github.com/pydanny/cached-property/issues/69)
> 
> Looks like there's a work-around...

Uh oh, I foresee us forking and maintaining `cached_property` in the future, looking at that maintenance history...

But yay that there's a solution! https://github.com/pydanny/cached-property/pull/137

**Pull request merged**